### ISSUE TITLE
legibility & colour contrast

### ISSUE THREAD

**Issue by chungkky** [LibraryCarpentry/Top-10-FAIR#29]
I'm a recent visitor to your Top-10-FAIR page.  I notice the colour contrast is very low (blue text for links on a white background and light grey text on white background). It is hard to read and probably impossible for individuals with low vision.  The blue links do not met the minimum W3C WAG (Web Accessibility Guidelines Section 1.4.3) for legibility and colour contrast. 

WAG: https://webaim.org/standards/wcag/checklist#sc1.4.3
WAG: https://www.w3.org/TR/WCAG21/#contrast-minimum

If you want to verify this for yourselves, there is a well respected web browser plugin tool for checking website for accessibility called WAVE (accessibility evaluation tool web), created by Utah State University, here https://wave.webaim.org/

I also find the grey text on a white background hard to read (even though it seems to pass the WAG standards).  Please consider making that text darker also.

Thank you.

**Comment by libcce** [comment]
Thank you @chungkky! We can definitely make these changes. Before I do, do you have additional suggestions per color/contrast? Otherwise, I can increase to a darker blue for links and a darker black from gray for the subheadings, body text.

**Comment by chungkky** [comment]
Hi @libcce, The short answer is: darker blue for the links and darker grey or even black for the text do the trick. :-)  Though note; I'm not an accessibility expert, just a person with serious myopia who is now closer to 60 than 50! 

In general, higher contrast is better for legibility.  Just compare the GitHub Octocat logo in the footer and the Octocat logo in the header of this page!  

Also, for a given colour contrast, the larger the font size and the "thicker" the strokes on the font type, the easier it is to see.  

If you are really serious, there are also some colour combinations which you might also avoid for different kinds of colour-blindness but I'm not an expert on that issue.  If you want know more, these seems like good places to start: 

- [https://www.w3.org/WAI/](https://www.w3.org/WAI/)
- [https://www.w3.org/WAI/ER/tools/](https://www.w3.org/WAI/ER/tools/)
- [https://accessibility.psu.edu/color/contrasthtml/](https://accessibility.psu.edu/color/contrasthtml/).  

I use browser plugins to do quick checks on webpages for WCAG standards.  For Firefox, there is WAVE (as in my previous note) and a plugin called WCAG Contrast Checker.

Cheers.
